File: Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_hashes(int n);
void print_spaces(int n);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);

    for (int i = height; i > 0; i--)
    {
        print_spaces(i - 1);
        print_hashes(height - i + 1);
        printf("  ");
        print_hashes(height - i + 1);
        printf("\n");
    }
}

void print_hashes(int n)
{
    for (int j = 1; j <= n; j++)
    {
        printf("#");
    }
}

void print_spaces(int n)
{
    for (int j = n; j > 0; j--)
    {
        printf(" ");
    }
}

File: Week 6/sentimental-mario-more/mario.py
from cs50 import *


def main():

    height = get_int("Height: ")

    while height < 1 or height > 8:
        print("Height must be a value between 1 and 8 (inclusive)")
        height = get_int("Height: ")

    for i in range(1, height + 1):
        print(" " * (height - i), end="")
        print("#" * i, end="")
        print("  ", end="")
        print("#" * i)


if __name__ == "__main__":
    main()
